### DISCUSSION TITLE
More replay functionality

### DISCUSSION THREAD

**Discussion by crumpledmint** [ppy/osu#28945] [Ideas] [answered]
Being able to fast forward, rewind and set a pause are already steps in the right direction for helping people to understand where and why they missed but it would have been cool if there also were some functions of "Rewind" tool such as timeline with misses and keypresses timeline for a more detailed understanding of why you missed on a certain part. Also "H" keybind from the stable to hide the UI of the replay would have been nice too

**Comment by Joehuu** [comment, marked as answer]
> timeline with misses

Already tracked in https://github.com/ppy/osu/issues/5681.

> keypresses timeline

There is already a pending change in https://github.com/ppy/osu/pull/27334. Original discussion is in https://github.com/ppy/osu/discussions/26856.

> Also "H" keybind from the stable to hide the UI of the replay would have been nice too

It's `Ctrl-H` now.

**Reply by crumpledmint** [reply]
I don't think it covers the inputs timeline (as in rewind), should I add it to the replies in that discussion?

**Reply by Joehuu** [reply]
Can just make another discussion. Preferably we want one feature per discussion, so it is easier to track and upvotes are more accurate.
